client/my-sites/email/email-management/home/email-mailbox-warnings.jsx

```diff
@@ -0,0 +1,167 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { Button } from '@automattic/components';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'calypso/components/gridicon';
+import { resendVerificationEmail } from 'calypso/state/email-forwarding/actions';
+import {
+	getEmailForwardAddress,
+	hasGoogleAccountTOSWarning,
+	isEmailUserAdmin,
+	isGoogleEmailAccount,
+} from 'calypso/lib/emails';
+import { getGoogleAdminUrl } from 'calypso/lib/gsuite';
+import { recordTracksEvent } from 'calypso/state/analytics/actions';
+import {
+	EMAIL_WARNING_SLUG_GOOGLE_MAILBOX_TOS,
+	EMAIL_WARNING_SLUG_UNVERIFIED_FORWARD,
+} from 'calypso/lib/emails/email-provider-constants';
+import { isEmailForwardAccount } from 'calypso/lib/emails/is-email-forward-account';
+
+const MailboxListItemWarning = ( { warningText } ) => {
+	return (
+		<div className="email-mailbox-warnings__warning">
+			<Gridicon icon="info-outline" size={ 18 } />
+			<span>{ warningText }</span>
+		</div>
+	);
+};
+
+const MailboxListItemAction = ( { buttonText, external, ...otherProps } ) => {
+	return (
+		<div className="email-mailbox-warnings__action">
+			<Button compact { ...otherProps }>
+				<span>{ buttonText }</span>
+				{ external && <Gridicon icon="external" /> }
+			</Button>
+		</div>
+	);
+};
+
+const resendEmailForwardVerification = ( mailbox, dispatch ) => {
+	const destination = getEmailForwardAddress( mailbox );
+	dispatch(
+		recordTracksEvent(
+			'calypso_email_management_email_forwarding_resend_verification_email_click',
+			{
+				destination,
+				domain_name: mailbox.domain,
+				mailbox: mailbox.mailbox,
+			}
+		)
+	);
+	dispatch( resendVerificationEmail( mailbox.domain, mailbox.mailbox, destination ) );
+};
+
+const getDetailsForWarning = ( { account, dispatch, mailbox, translate, warning } ) => {
+	const warningSlug = warning.warning_slug;
```

Comment:
This (is just to show off my Javascript skillz 😆 but this) could also be written like this:

```javascript
	const { warning_slug: warningSlug } = warning;
```

Reply:
For this case, I feel that's more verbose and doesn't buy us much. I'll leave this as-is for now.